### DISCUSSION TITLE
More modern ui for systrem tray menu.

### DISCUSSION THREAD

**Discussion by MikaelAarras** [Rem0o/FanControl.Releases#2301] [Ideas]
Small change but would be appreciated. 
Basically this would just change the look of the right click menu in the system tray icon. At the moment it follows quite old design at least in my opinion. The new windows default right click menu has rounded corners and softer colors. I'm sure people get what I'm saying.

**Comment by Rem0o** [comment]
Yeah the API I'm using to get that tray icon is quite old (WinForm) 😅. There isn't much alternatives available AFAIK.
